### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by vvtimofeev**
Updated readme file: added UIKit picture in the beginning, some links to our resources, slightly reformulated UIKit description phrase.

### CODE REVIEW

File: docs/assets/logo.svg
Comment (amje) [inline docs/assets/logo.svg:1]: Seems not used

**Review by amje — CHANGES_REQUESTED** [review]

**Review by korvin89 — CHANGES_REQUESTED** [review]
I suppose we should wait for this feature https://github.com/gravity-ui/landing/issues/200 to have opportunity to hide this content in landing page

**Comment by vvtimofeev** [comment]
> I suppose we should wait for this feature [gravity-ui/landing#200](https://github.com/gravity-ui/landing/issues/200) to have opportunity to hide this content in landing page

I added special markdown block <GITHUB_BLOCK>, it should hide the image and the resources block from the landing. If you don't mind, please reconsider your comment.

**Review by korvin89 — APPROVED** [review]

**Comment by korvin89** [comment]
I suppose that icons will be poorly distinguishable in a dark theme

**Review by amje — APPROVED** [review]

**Pull request merged**